src/plugins/blocksmith/components/image-block-modal/CustomizeOptions.jsx

```diff
@@ -1,18 +1,22 @@
+import React from 'react';
+import { Field } from 'redux-form';
+import PropTypes from 'prop-types';
+
 import AspectRatioEnum from '@triniti/schemas/triniti/common/enums/AspectRatio';
-import DateTimePicker from '@triniti/cms/plugins/blocksmith/components/date-time-picker';
 import humanizeEnums from '@triniti/cms/utils/humanizeEnums';
 import ImageAssetPicker from '@triniti/cms/plugins/dam/components/image-asset-picker';
 import ImageBlockPreview from '@triniti/cms/plugins/blocksmith/components/image-block-preview';
 import Message from '@gdbots/pbj/Message';
-import PropTypes from 'prop-types';
-import React from 'react';
 import UncontrolledTooltip from '@triniti/cms/plugins/common/components/uncontrolled-tooltip';
+import PicklistPickerField from '@triniti/cms/plugins/sys/components/picklist-picker-field';
 import {
   Checkbox,
   FormGroup,
   Icon,
   Input,
-  Label,
```

Comment:
@jalopez13 - you are using the `Label` component, please add it back

Reply:
@arunkarnati 

Added Label component.